sdk/resource/detect/env.go

```diff
@@ -0,0 +1,62 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package detect provides a detector that loads resource information from
+// the OTEL_RESOURCE environment variable. A list of labels of the form
+// `<key1>=<value1>,<key2>=<value2>,...` is accepted. Domain names and
+// paths are accepted as label keys.
+package detect
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"go.opentelemetry.io/otel/api/kv"
+	"go.opentelemetry.io/otel/sdk/resource"
+)
+
+// Environment variable used by "env" to decode a resource.
+const envVar = "OTEL_RESOURCE"
```

Comment:
The comment from the previous PR seems to still be valid: https://github.com/open-telemetry/opentelemetry-go/pull/924#discussion_r454553037

Is there any update here?

I'm pretty sure based on https://github.com/open-telemetry/opentelemetry-specification/pull/666 currently this needs to be `OTEL_RESOURCE_LABELS`, though it points to https://github.com/open-telemetry/oteps/blob/master/text/0111-auto-resource-detection.md which has it this way.